src/response.rs

```diff
@@ -263,6 +263,14 @@ impl Into<http::Response> for Response {
     }
 }
 
+impl From<serde_json::Value> for Response {
+    fn from(json_value: serde_json::Value) -> Self {
+        Response::new(StatusCode::Ok)
```

Comment:
One thing about this -- it'd be nice to have non-200 status codes supported for easy json responses.

Is it also possible to do something like `impl From<(StatusCode, serde_json::Value)>`?

Reply:
I mean it's _possible_ to provide
```
impl<S: TryInto<StatusCode>, R: Into<Response>> From<(S, R)> for Response {
    fn from(tuple: (S, R)) -> Self {
        tuple.1.into().set_status(
            tuple
                .0
                .try_into()
                .unwrap_or_else(|_| panic!("Invalid status code")),
        )
    }
}
```

But I don't have a great sense of whether it's good design.  That would let you do like `Ok(200, json!({"status": "ok"}))` or `Ok((StatusCode::NotFound, "not found"))`, etc

Reply:
Another option to make constructing responses even easier would be to provide a `respond!` macro that can take variadic args and is a shortcut for building an `Ok(Response::new(…))`

Reply:
The way I was thinking that should be done instead is by replying on the `Response:: new` constructor:

```async fn respond(_: Request) -> Result<Response> {
    let mut res = Response::new(203);
    res.set_body(json!({"color": "cyan"});
    Ok(res)
}
```
I don't feel converting from a tuple or introducing a new macro would improve much on this.